docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -64,6 +64,12 @@ Bugs
 Known Issues
 ------------
 
+* CentOS 7 users may experience a problem when upgrading. Please refer to note in upgrade
+  instructions for workaround.
+
+* Users that have `/var/lib/pulp` directory symlinked will experience a
+  `problem <https://pulp.plan.io/issues/1791>`_. Bind mounting this directory will resolve the issue.
```

Comment:
s/Bind mounting this directory/Replacing the symlink with an equivalent bind mount"